.azure-pipelines-templates/cmake.yml

```diff
@@ -14,9 +14,3 @@ steps:
   env:
     TESTS_SUFFIX: ${{ parameters.suite_label_suffix }}
   displayName: CMake
-
```

Comment:
This was split because I was hoping to re-use the install bundle produced by the Release job (but would still need to re-run cmake to get ctest configured). In the end it wasn't to be, because we don't install some binaries we need for the perf tests (for good reasons). But this will probably still happen, so I'm keeping them split.